lib/tapioca/compilers/dsl/smart_properties.rb

```diff
@@ -127,7 +127,10 @@ def generate_methods_for_property(klass, property, &block)
           klass.create_method(property.reader.to_s, return_type: type) if block.call(property.reader.to_s)
         end
 
-        BOOLEANS = [[true, false], [false, true]].freeze
+        BOOLEANS = T.let([
+          [true, false],
+          [false, true]
+        ].freeze, T::Array[[T::Boolean, T::Boolean]])
```

Comment:
is this change just for making it more secure? or was there an actual issue that this change is fixing

Reply:
No, this is necessary for `typed: strict`. At that strictness level, all constants must have their types declared using `T.let`.